tests/src/Behat/OeThemeTestContext.php

```diff
@@ -398,8 +408,65 @@ public function assertEuMobileLogo(string $language): void {
       'Finnish' => 'fi',
       'Swedish' => 'sv',
     ];
+    $langcode = $lang_code[$language] ?? 'en';
     $this->assertSession()->elementExists('css', 'img.ecl-site-header-' . $this->getEclBranding() . '__logo-image-mobile');
-    $this->assertSession()->elementAttributeContains('css', 'img.ecl-site-header-' . $this->getEclBranding() . '__logo-image-mobile', 'src', 'oe_theme/dist/eu/images/logo/condensed-version/positive/' . $lang_code[$language] . '.svg');
+    $this->assertSession()->elementAttributeContains('css', 'img.ecl-site-header-' . $this->getEclBranding() . '__logo-image-mobile', 'src', 'oe_theme/dist/eu/images/logo/condensed-version/positive/logo-eu--' . $langcode . '.svg');
+  }
+
+  /**
+   * Asserts that the header logo contains all required attributes.
+   *
+   * @Then the :language header logo should contain accessibility attributes
+   */
+  public function assertHeaderLogoAccessibility(string $language): void {
+    $theme_name = \Drupal::theme()->getActiveTheme()->getName();
+    $component_library = \Drupal::config($theme_name . '.settings')->get('component_library');
+    $eu_official_languages = [
```

Comment:
Can we use oe_theme/modules/oe_theme_helper/src/EuropeanUnionLanguages.php as a single entry point for EU languages ?